public/app/core/reducers/navModel.test.ts

```diff
@@ -38,4 +38,26 @@ describe('applicationReducer', () => {
         });
     });
   });
+
+  describe('when updateConfigurationSubtitle is dispatched', () => {
+    it('then state should be correct', () => {
+      const orgName = 'New Org Name';
+      const newCfgState = { ...initialState.cfg, subTitle: `Organization: ${orgName}` };
+      const expectedState = {
+        ...initialState,
+        cfg: newCfgState,
+        datasources: { ...initialState.datasources, parentItem: newCfgState },
+        users: { ...initialState.users, parentItem: newCfgState },
+        teams: { ...initialState.teams, parentItem: newCfgState },
+        plugins: { ...initialState.plugins, parentItem: newCfgState },
+        ['org-settings']: { ...initialState['org-settings'], parentItem: newCfgState },
```

Comment:
```suggestion
        'org-settings': { ...initialState['org-settings'], parentItem: newCfgState },
```
Square brackets are normally used for calculated properties, which is not the case.